src/Umbraco.Core/PropertyEditors/ValueConverters/SliderValueConverter.cs

```diff
@@ -11,77 +8,73 @@ namespace Umbraco.Core.PropertyEditors.ValueConverters
     [DefaultPropertyValueConverter]
     public class SliderValueConverter : PropertyValueConverterBase
     {
-        private readonly IDataTypeService _dataTypeService;
+        public SliderValueConverter()
+        { }
 
+        [Obsolete("This constructor isn't required anymore, because we don't need services to lookup the data type configuration.")]
         public SliderValueConverter(IDataTypeService dataTypeService)
-        {
-            _dataTypeService = dataTypeService ?? throw new ArgumentNullException(nameof(dataTypeService));
-        }
+        { }
 
         public override bool IsConverter(IPublishedPropertyType propertyType)
             => propertyType.EditorAlias.InvariantEquals(Constants.PropertyEditors.Aliases.Slider);
 
         public override Type GetPropertyValueType(IPublishedPropertyType propertyType)
-            => IsRangeDataType(propertyType.DataType.Id) ? typeof (Range<decimal>) : typeof (decimal);
+            => IsRangeDataType(propertyType) ? typeof(Range<decimal>) : typeof(decimal);
```

Comment:
This PVC can also return `null` (for no or an invalid value), so the type for single values should actually be `typeof(decimal?)`, right? This will probably be a breaking change, so I haven't included it in this PR.

Although changing the value type is done previously for the Media Picker (https://github.com/umbraco/Umbraco-CMS/issues/6033), it's not without problems, as it changes the generated models and all code that expects a non-nullable value.